js/src/wrappers/oai.ts

```diff
@@ -343,7 +361,7 @@ function parseBaseParams<T extends Record<string, any>>(
 ): StartSpanArgs {
   const { span_info, ...params } = allParams;
   const { metadata: spanInfoMetadata, ...spanInfoRest } = span_info ?? {};
-  let ret: StartSpanArgs = {
+  const ret: StartSpanArgs = {
```

Comment:
i wonder if there's a lint rule for this also.

Reply:
there is :) that's why my IDE asked to change it